From stern@rowland.harvard.edu Tue Jan 30 07:34:18 2007
From: Alan Stern <stern@rowland.harvard.edu>
Date: Tue, 30 Jan 2007 10:34:15 -0500 (EST)
Subject: USB: Fix fallout from usbcore bandwidth removal
To: Greg KH <greg@kroah.com>
Message-ID: <Pine.LNX.4.44L0.0701301031090.3473-100000@iolanthe.rowland.org>


This patch (as845) cleans up some fallout from the removal of the
usbcore bandwidth machinery.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/isdn/gigaset/bas-gigaset.c |    4 ++--
 sound/usb/usbaudio.c               |   10 ----------
 2 files changed, 2 insertions(+), 12 deletions(-)

--- gregkh-2.6.orig/drivers/isdn/gigaset/bas-gigaset.c
+++ gregkh-2.6/drivers/isdn/gigaset/bas-gigaset.c
@@ -257,10 +257,10 @@ static inline void dump_urb(enum debugle
 			urb->transfer_flags);
 		gig_dbg(level,
 			"  transfer_buffer=0x%08lx[%d], actual_length=%d, "
-			"bandwidth=%d, setup_packet=0x%08lx,",
+			"setup_packet=0x%08lx,",
 			(unsigned long) urb->transfer_buffer,
 			urb->transfer_buffer_length, urb->actual_length,
-			urb->bandwidth, (unsigned long) urb->setup_packet);
+			(unsigned long) urb->setup_packet);
 		gig_dbg(level,
 			"  start_frame=%d, number_of_packets=%d, interval=%d, "
 			"error_count=%d,",
--- gregkh-2.6.orig/sound/usb/usbaudio.c
+++ gregkh-2.6/sound/usb/usbaudio.c
@@ -325,16 +325,6 @@ static int prepare_capture_urb(struct sn
 	}
 	urb->transfer_buffer_length = offs;
 	urb->number_of_packets = ctx->packets;
-#if 0 // for check
-	if (! urb->bandwidth) {
-		int bustime;
-		bustime = usb_check_bandwidth(urb->dev, urb);
-		if (bustime < 0)
-			return bustime;
-		printk("urb %d: bandwidth = %d (packets = %d)\n", ctx->index, bustime, urb->number_of_packets);
-		usb_claim_bandwidth(urb->dev, urb, bustime, 1);
-	}
-#endif // for check
 	return 0;
 }
 
